### PR TITLE
Update on the Brazilian Portuguese localization

### PR DESCRIPTION
**Pull request by E-rick23**
While using the software, i noticed a few parts were still using the default language (English), so, i added the translation to those parts (most of changes are on the software settings), i hope it's helpful.

### CODE REVIEW

File: Cairo Desktop/CairoDesktop.Localization/Language.cs [hunk 1]
```diff
@@ -486,6 +504,17 @@ static class Language
             { "sSettings_Taskbar_DisplayModeOverlap", "Barra de tarefas sobrepõe janelas"},
             { "sSettings_Taskbar_DisplayModeAutoHide", "Barra de tarefas oculta automaticamente"},
             { "sSettings_Taskbar_EnableTaskbarMultiMon", "Mostrar barra de tarefas em vários monitores"},
+            { "sSettings_Taskbar_FullWidthTaskBar", "Usar a largura total da tela para a barra de tarefas"}
```
Comment (dremin) [inline Cairo Desktop/CairoDesktop.Localization/Language.cs:507]: This line is missing a comma, could you please fix that?

**Review by dremin — CHANGES_REQUESTED** [review]
Thanks for the update! There's an issue preventing the build (missing comma), could you please fix that?

File: Cairo Desktop/CairoDesktop.Localization/Language.cs (hunk 1 above)
Comment (E-rick23) [reply Cairo Desktop/CairoDesktop.Localization/Language.cs]: There, i think it should be fixed now, sorry about that.

**Review by dremin — APPROVED** [review]
Thanks!

**Pull request merged**